kex-runner/src/main/kotlin/org/vorpal/research/kex/asm/analysis/symbolic/InstructionSymbolicChecker.kt

```diff
@@ -17,10 +17,22 @@ class InstructionSymbolicChecker(
     ctx: ExecutionContext,
     rootMethod: Method,
 ) : SymbolicTraverser(ctx, rootMethod) {
-    override val pathSelector: SymbolicPathSelector = DequePathSelector()
+    override val pathSelector: SymbolicPathSelector
     override val callResolver: SymbolicCallResolver = DefaultCallResolver(ctx)
     override val invokeDynamicResolver: SymbolicInvokeDynamicResolver = DefaultCallResolver(ctx)
 
+    init {
+        val pathSelectorName: String = kexConfig.getStringValue("kex", "pathSelector", "NSubpathPathSelector")
+        val n: Int = kexConfig.getIntValue("kex", "nPathSelectorArg", 2)
+        pathSelector = when (pathSelectorName) {
+            "DequePathSelector" -> DequePathSelector()
```

Comment:
I would prefer using underlying algorithm names to differentiate different path selectors. E.g. "bfs", "sgs" (subpath guided search, as it is called in the paper), "priority-bfs". It is easier to remebmer